blob/example_test.go

```diff
@@ -329,7 +356,7 @@ func ExampleListObject_As() {
 	}
 }
 
-func ExampleListOptions_BeforeList() {
+func ExampleListOptions() {
```

Comment:
I changed this because godoc currently doesn't show this example at all. I suspect this is because `BeforeList` is a struct field rather than a method/function. With the new name it will hopefully just show beneath the struct.

Reply:
ok